routes/web.php

```diff
@@ -16,9 +17,16 @@
 */
 
 
-Route::get('/', function () {
-    return view('welcome');
-});
+// Route::get('/', function () {
+//     return view('welcome');
+// });
+Route::get('/',[HomeController::class,'index'])->name('home');
+
+Route::get('/login', [UserController::class, 'login'])->name('login');
+Route::post('/login', [UserController::class, 'postLogin'])->name('login.post');
+Route::get('/logout', [UserController::class, 'logout'])->name('logout');
+Route::get('/register', [UserController::class, 'register'])->name('register');
+Route::post('/register', [UserController::class, 'postRegister'])->name('register.post');
```

Comment:
```suggestion
Route::get('/register', [UserController::class, 'register'])->name('register.form');
Route::post('/register', [UserController::class, 'postRegister'])->name('register');
```
em nên đặt tên đúng action cho dễ hiểu nhé, còn ghi `register.post` có thể ng khác hiểu nhưng nó không rõ ràng lắm